package.json

```diff
@@ -1,45 +1,53 @@
 {
   "name": "platform-app-coinify",
-  "version": "0.1.0",
+  "version": "0.2.0",
   "private": true,
+  "engines": {
+    "node": ">=18.17"
+  },
   "scripts": {
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
+    "clean": "rm -rf node_modules .next",
     "format:check": "prettier --check 'src/**' 'pages/**'",
     "format:fix": "prettier --write 'src/**' 'pages/**'",
     "lint:check": "eslint --cache src/** pages/**",
     "lint:fix": "eslint --cache --fix src/** pages/**"
   },
   "dependencies": {
-    "@ledgerhq/live-app-sdk": "0.6.0",
-    "@ledgerhq/react-ui": "^0.7.2",
+    "@babel/core": "^7.23.9",
+    "@ledgerhq/exchange-sdk": "file:./../exchange-sdk/lib",
```

Comment:
can we change this to a remote repo? 🤔

Reply:
Yes definitely.
I will update this PR as soon as we got a new release of the ExchangeSDK, including Sell capability :)